### DISCUSSION TITLE
Monterey Guidance Revision 4

### DISCUSSION THREAD

**Discussion by robertgendler** [usnistgov/macos_security#272] [General]
Included in this release are updated guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, DISA-STIG, NIST 800-171, CIS macOS Benchmarks Level 1 and 2, and CIS Critical Security Controls Version 8 baselines for macOS Monterey (12.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/monterey_rev4'>Monterey Guidance Revision 4</a>.</em>

**Comment by ghost** [comment]
in this line /bin/launchctl unload /System/Library/LaunchDaemons/com.apple.ODSAgent.plist should there be a -w after unload to make the change permanent?

**Reply by robertgendler** [reply]
Maybe?

Though it's set to be disabled by default so there's no reason to double disable it. It only really launches when you set up cd/dvd sharing. It never loads at boot on its own.

**Comment by ghost** [comment]
Yep. I was more thinking for consistency which is key.

:-)

-Vaughn

2023년 6월 27일 (화) 오후 2:07, Bob Gendler ***@***.***>님이 작성:

> Maybe?
>
> Though it's set to be disabled by default so there's no reason to double
> disable it. It only really launches when you set up cd/dvd sharing. It
> never loads at boot on its own.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/usnistgov/macos_security/discussions/272#discussioncomment-6296774>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYOKYHRWCEPDU7FDFPJLLGDXNMOM3ANCNFSM6AAAAAAZUTUXQM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
-- 


---------------------------------
Vaughn A. Hart
646-284-4291
***@***.*** ***@***.***>
www.linkedin.com/in/vahart
https://github.com/vaughnhart
